src/site/filters/liquid.js

```diff
@@ -593,4 +589,47 @@ module.exports = function registerFilters() {
 
   liquid.filters.sortEntityMetatags = item =>
     item ? item.sort((a, b) => a.key.localeCompare(b.key)) : undefined;
+
+  liquid.filters.createEmbedYouTubeVideoURL = url => {
+    if (!url) {
+      return url;
+    }
+
+    if (!_.includes(url, 'youtu')) {
+      return url;
+    }
+
+    if (_.includes(url, 'embed')) {
+      return url;
+    }
+
+    return _.replace(url, 'youtu.be', 'youtube.com/embed');
+  };
+
+  liquid.filters.formatSeconds = rawSeconds => {
+    // Dates need milliseconds, so mulitply by 1000.
+    const date = new Date(rawSeconds * 1000);
+
+    // Derive digits.
+    const hours = date.getUTCHours() || '';
+    const minutes = date.getUTCMinutes() || '';
+    const seconds = date.getUTCSeconds() || '';
```

Comment:
I wonder if we could get away with using Moment for this like `moment.unix(rawSeconds).format('hh:mm:ss')` -> `01:07:25`

Reply:
Nic with the clutch moment api knowledge haha.

Reply:
That's one way to do it! We'd have to clean the `00:00:0`s when there aren't seconds, so if it's ok I'll stick with my current solution 🙂